addons/website_sale/controllers/main.py

```diff
@@ -208,6 +217,10 @@ def _get_search_domain(self, search, category, attrib_values, search_in_descript
             if attrib:
                 domains.append([('attribute_line_ids.value_ids', 'in', ids)])
 
+        extra_domain = self._add_search_domains_hook(search, category, attrib_values)
+        if extra_domain:
+            domains.append(extra_domain)
+
```

Comment:
This second hook doesn't seem very useful to me.
Can't you get the same result with an override of `_get_search_domain` instead ?
```python
def _get_search_domain(...):
    res = super()._get_search_domain(...)
    return AND([res, your_custom_domain])
```

Reply:
@Feyensv Thanks for the review.

I guess you're right here, as the most important hook for what we did was the one on subdomains.